### DISCUSSION TITLE
Debug Message For Projectile

### DISCUSSION THREAD

**Discussion by AriaMaii** [Pryaxis/TShock#2019] [Q&A]
Good afternoon,

I have been using the last version of Tshock to host a server just for me and a few friends on Journey Mode.

I downloaded the new Tshock launcher yesterday and started up my server, but I keep getting the messages attached below every time I open a door, or use a weapon. I'm not sure what is causing this. The previous version of Tshock never gave me this kind of message.

![image](https://user-images.githubusercontent.com/66568843/83976953-55d3e900-a8cb-11ea-8a75-5b4a525a32a3.png)

**Comment by hakusaro** [comment]
This is pre 10?

**Reply by AriaMaii** [reply]
It's pre 11
